app/models/hall.rb

```diff
@@ -0,0 +1,4 @@
+class Hall < ApplicationRecord
+  validates :number, :capacity, presence: true
```

Comment:
are you sure it should be the number? why not name? what if we want to name halls: red, white, or blue?

Reply:
good point, I will change it to string